src/dc_configure.rs

```diff
@@ -194,16 +194,15 @@ pub unsafe fn dc_job_do_DC_JOB_CONFIGURE_IMAP(context: &Context, _job: *mut dc_j
                                     }) as uintptr_t,
                                     0 as uintptr_t,
                                 );
-                                current_block = 7746103178988627676;
+                                ok_to_continue0 = true;
```

Comment:
Same thing as with the other pr, initializing it as a true should be fine.

Reply:
I plan to try refactoring this file to make more sense soon anyway

Reply:
Fine than lets merge it as it is